graphql/util.go

```diff
@@ -561,10 +567,29 @@ func generateBuildVariants(ctx context.Context, sc data.Connector, versionId str
 		}
 		result = append(result, &pbv)
 	}
+
+	timeToGroupTasks := time.Since(groupTasksStartTime)
+
+	sortTasksStartTime := time.Now()
 	// sort variants by name
 	sort.SliceStable(result, func(i, j int) bool {
 		return result[i].DisplayName < result[j].DisplayName
 	})
+
+	timeToSortTasks := time.Since(sortTasksStartTime)
+
+	totalTime := time.Since(start)
+	grip.InfoWhen(totalTime > time.Second*2, message.Fields{
+		"Ticket":            "EVG-14828",
+		"timeToFindTasks":   timeToFindTasks,
```

Comment:
times are stored in milliseconds; I often find it easier to read when converted to seconds